library/patient.inc.php

```diff
@@ -630,7 +668,7 @@ function getPatientId($pid = "%", $given = "pid, id, lname, fname, mname, provid
 function getByPatientDemographics($searchTerm = "%", $given = "pid, id, lname, fname, mname, providerID, DATE_FORMAT(DOB,'%m/%d/%Y') as DOB_TS", $orderby = "lname ASC, fname ASC", $limit = "all", $start = "0")
 {
     $layoutCols = sqlStatement(
-        "SELECT field_id FROM layout_options WHERE form_id = 'DEM' AND field_id not like ? AND uor !=0",
+        "SELECT field_id FROM layout_options WHERE form_id = 'DEM' AND field_id not like ? AND uor ! = 0",
```

Comment:
can revert or could be 
```
"SELECT field_id FROM layout_options WHERE form_id = 'DEM' AND field_id not like ? AND uor != 0",
```

Reply:
@stephenwaite have you completed your testing?

Reply:
hi @juggernautsei, is it okay if I test after you push the other suff?

Reply:
Yes

Reply:
hi @juggernautsei, if you'd like I can push the changes here to try and help with that [community forum thread](https://community.open-emr.org/t/patient-insurance-history-build/17416/19?u=stephenwaite).

Reply:
yes, lets do that